inst/NEWS.Rd

```diff
@@ -3,6 +3,7 @@
 \newcommand{\cpkg}{\href{http://CRAN.R-project.org/package=#1}{\pkg{#1}}}
 \section{Changes in version 2.8-4 (GitHub-only-release date:?-?-2022)}{
   \itemize{
+    \item Part 5: dayborder != 0 is now handled in g.part5.wakesleepwindow when nonwear is high
```

Comment:
Hi @jhmigueles, it looks correct but I am not sure I understand why you state that the problem that is fixed is specific to the condition when non-wear is high?

If I understand correctly `nightsi` and `nightsi2` are not dependent on the amount of non-wear? How does nonwear affect all of this?